pubspec.lock

```diff
@@ -155,7 +162,7 @@ packages:
       name: vector_math
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "2.1.0"
+    version: "2.1.1"
 sdks:
-  dart: ">=2.12.0 <3.0.0"
+  dart: ">=2.14.0 <3.0.0"
```

Comment:
You have changed support for 2.12.0 to 2.14.0 so any specific reason? Please share.